test/Sanitizers/tsan/racy_async_let_fibonacci.swift

```diff
@@ -15,7 +15,7 @@
 // UNSUPPORTED: OS=watchos
 
 // rdar://86825277
-// UNSUPPORTED: CPU=arm64e
+// UNSUPPORTED: CPU=arm64 || CPU=arm64e
```

Comment:
Can you add a reference to [SR-15805](https://bugs.swift.org/browse/SR-15805) here?

Reply:
```suggestion
// SR-15805
// UNSUPPORTED: CPU=arm64 || CPU=arm64e
```